pymongo/asynchronous/mongo_client.py

```diff
@@ -803,7 +803,10 @@ def __init__(
         if tz_aware is None:
             tz_aware = opts.get("tz_aware", False)
         if connect is None:
-            connect = opts.get("connect", True)
+            # Default to connect=True unless on a FaaS system, which might use fork.
```

Comment:
Suggest adding a versionchanged and updating the docstring for `connect`.

Reply:
Done